src/widget/sidebar/common.h

```diff
@@ -0,0 +1,19 @@
+#ifndef WIDGET_SIDEBAR_COMMON_H
+#define WIDGET_SIDEBAR_COMMON_H
+
+#include "graphics/menu.h"
+
+#define SIDEBAR_COLLAPSED_WIDTH 42
+#define SIDEBAR_EXPANDED_WIDTH 162
+#define SIDEBAR_VANILLA_SECTION_HEIGHT 450
```

Comment:
I don't quite like the VANILLA name. Perhaps rename this to `SIDEBAR_MAIN_SECTION_HEIGHT`?